node/src/reactor/participating.rs

```diff
@@ -1141,6 +1340,12 @@ impl reactor::Reactor for Reactor {
                     linear_chain::Event::GotUpgradeActivationPoint(next_upgrade.activation_point()),
                 );
                 effects.extend(self.dispatch_event(effect_builder, rng, reactor_event));
+                let reactor_event = ParticipatingEvent::ChainSynchronizer(
+                    chain_synchronizer::Event::GotUpgradeActivationPoint(
+                        next_upgrade.activation_point(),
+                    ),
+                );
```

Comment:
(Will this ever be relevant for the chain synchronizer in participating mode?)

Reply:
Umm - looks like this entirely unused by the chain synchronizer in both reactors.  I'll look into this, but I suspect we can just remove it altogether.  Good catch! :)